sdk/communication/Azure.Communication.CallingServer/src/CallingServerClientOptions.cs

```diff
@@ -27,7 +27,7 @@ public CallingServerClientOptions(ServiceVersion version = LatestVersion)
         {
             ApiVersion = version switch
             {
-                ServiceVersion.V2021_06_15_Preview => "2021-06-15-preview",
```

Comment:
Should we keep the older values in addition to the new values in the switch?

Reply:
That makes sense, Let me keep both. But "LatestVersion" could be just single version. That needs to be the latest.